app/backend/chat_history/cosmosdb.py

```diff
@@ -150,10 +192,22 @@ async def delete_chat_history_session(auth_claims: Dict[str, Any], item_id: str)
         return jsonify({"error": "User OID not found"}), 401
 
     try:
-        await container.delete_item(item=item_id, partition_key=entra_oid)
-        return jsonify({}), 204
+        res = container.query_items(
+            query="SELECT * FROM c WHERE c.session_id = @session_id",
```

Comment:
Do we need to select * if we are just going to delete all the items? can we just get the id?

Reply:
Good point! I copied from above. I'll change to just id.